resources/views/index.blade.php

```diff
@@ -1,85 +1,85 @@
 @extends('layout.master')
 
 @section('content')
-    @if($subscribersEnabled)
-    <div class="pull-right">
-        <p><a class="btn btn-success btn-outline" href="{{ route('subscribe.subscribe') }}">{{ trans('cachet.subscriber.button') }}</a></p>
-    </div>
-    @endif
+@if($subscribersEnabled)
```

Comment:
Oh yeah... I'm not sure why that's like it actually, but I'll fix it on this branch.